### ISSUE TITLE
CSS preserve not working

### ISSUE THREAD

**Issue by jackocnr** [prettydiff/prettydiff#274]
Go to http://prettydiff.com

Enter the following CSS:
```css
.a {
  margin: 0;

  padding: 10px;
}
```

Make sure "Preserve empty lines" is checked and click Execute.

Expected result:
```css
.a {
  margin: 0;

  padding: 10px;
}
```

Actual result:
```css
.a {
  margin: 0;
  padding: 10px;
}
```

**Comment by prettydiff** [comment]
This is resolved in v1.16.22.  It was more an enhancement than a defect as I never really supported empty lines prior to a property name, because it can sometimes end up strange after sorting.

**Comment by jackocnr** [comment]
Tested and confirmed, thank you.
